tmt/steps/__init__.py

```diff
@@ -265,6 +267,16 @@ def __init__(self, step, data):
                 f"Missing 'name' in the {step} step config "
                 f"of the '{step.plan}' plan.")
 
+        # FIXME Convert boolean keys 'True' into string 'on'
+        # This is necessary until python3-ruamel-yaml >= 0.16
+        # is available on epel-8 as well as it seems that 0.15
+        # does not fully support the yaml 1.2 version.
+        # https://bugzilla.redhat.com/show_bug.cgi?id=1700334
+        try:
+            data["on"] = data.pop(True)
```

Comment:
This can be removed, right? AFAIK attribute was renamed

Reply:
Yes. Hm, it got somehow lost during the rebase/squash.